outport/process/transactionsfee/transactionsFeeProcessor.go

```diff
@@ -328,6 +344,17 @@ func (tep *transactionsFeeProcessor) prepareScrsNoTx(transactionsAndScrs *transa
 			continue
 		}
 
+		isRelayedV3 := len(txFromStorage.InnerTransactions) > 0
+		if isRelayedV3 {
+			gasUnits := tep.txFeeCalculator.ComputeGasUnitsFromRefundValue(txFromStorage, scr.Value, epoch)
+
+			scrHandler.GetFeeInfo().SetForRelayed()
+			scrHandler.GetFeeInfo().SetGasUsed(gasUnits)
```

Comment:
can you add a new field for gasUnitsRefunded

Reply:
refactored